di-ipv-credential-issuer-stub/deploy/passport/template.yaml

```diff
@@ -52,6 +52,10 @@ Parameters:
     Description: "Environment Variables for ECS - Retrieved from SSM Parameter Store"
     Type: AWS::SSM::Parameter::Value<String>
     Default: "/stubs/credential-issuer/passport/env/VC_TTL_SECONDS" #pragma: allowlist secret
+  CredentialMitigationEnabled:
```

Comment:
As above

Reply:
Rename done - done for all the deploy templates.